projects/habitat_ovmm/configs/agent/heuristic_agent.yaml

```diff
@@ -56,7 +57,7 @@ SKILLS:
     min_joint_delta: 0.02
 
   PICK:
-    type: oracle
+    type: hw
```

Comment:
This won't work in sim. Today for me: fix this properly, so that pick works across sim/real without config changes